app/src/main/java/org/p2p/wallet/striga/offramp/mappers/StrigaOffRampSwapWidgetMapper.kt

```diff
@@ -75,7 +77,10 @@ class StrigaOffRampSwapWidgetMapper {
             isStatic = isStatic,
             widgetTitle = mapWidgetTitle(titleResId),
             availableAmount = mapAvailableAmount(isLoadingBalance, balance, currencyName),
-            currencyName = currencyName.toRawTextViewCellModel(R.style.UiKit_TextAppearance_Regular_Label1),
+            currencyName = currencyName.toRawTextViewCellModel(
+                textSizeSp = 28f,
```

Comment:
wow
Title1 should have text size already

Reply:
@gslevinkov it has, but nope, it just doesn't work despite SwapWidget [binds the model fully](https://github.com/p2p-org/key-app-android/blob/develop/ui-kit/src/main/kotlin/org/p2p/uikit/utils/text/TextViewCellModel.kt#L122)  and textViewSize by default is null, it uses some other text size, maybe directly from layout
![image](https://github.com/p2p-org/key-app-android/assets/130352737/18d67bb3-71c4-4d2e-a955-9139e8fca8ad)

Since we have textViewSize field, I honestly assumed that we can override this value directly in the model